blocks/share-bar-block/features/share-bar/default.jsx

```diff
@@ -70,25 +70,39 @@ function getLogoComponent(type) {
   }
 }
 
-const ShareBar = ({ useInjectedFusionContext = useFusionContext }) => {
+const ShareBarContainer = () => {
   const {
     customFields,
     globalContent: {
-      headlines = {},
+      headlines: { basic: headlineString = '' },
       website_url: websiteUrl = '',
     } = {},
     arcSite,
-  } = useInjectedFusionContext();
+  } = useFusionContext();
 
   const {
     websiteDomain,
     websiteName,
   } = getProperties(arcSite);
 
+  return (
+    <ShareBar
+      customFields={customFields}
+      websiteName={websiteName}
+      websiteDomain={websiteDomain}
+      websiteUrl={websiteUrl}
+      headlineString={headlineString}
+    />
+  );
+};
+
+export const ShareBar = ({
+  customFields, websiteName, websiteDomain, websiteUrl, headlineString,
+}) => {
   const shareButtons = [];
   Object.keys(customFields).forEach((social) => {
     if (customFields[social]) {
-      const encodedTitle = encodeURIComponent(headlines.basic);
+      const encodedTitle = encodeURIComponent(headlineString);
```

Comment:
passing in headlineString rather than the object that really only uses basic field